indexer-app/src/cmr/indexer/data/concepts/collection/humanizer.clj

```diff
@@ -34,15 +34,32 @@
                                 (add-humanized-lowercase value-with-priorities))]
     {field value-with-lowercases}))
 
+(defn humanized-field->elastic-doc
+  "Extracts humanized fields from the science keyword and places them into an elastic doc with
+  the same shape/keys as science-keyword->elastic-doc."
+  [field]
+  (let [humanized-fields (filter #(-> % key namespace (= "cmr.humanized")) field)
```

Comment:
Could you explain a bit what is going on here? I'm having trouble following.

Reply:
It's essentially just converting the humanized values to an elastic doc and associng the `.lowercase` values